sunpy/__init__.py

```diff
@@ -25,9 +25,10 @@
 from sunpy.util import system_info
 from sunpy.tests import main as self_test
 
-# Sample data
-from sunpy.data.sample import (AIA_171_IMAGE, RHESSI_IMAGE, EIT_195_IMAGE,
-                               RHESSI_EVENT_LIST, CALLISTO_IMAGE)
-
 # Load user configuration
 config = load_config()
+
+# Create shortcuts to sample data
+import sunpy.data.sample
+for key in sunpy.data.sample.sample_files:
+    setattr(sunpy.data.sample, key, sunpy.data.sample.sample_files[key])
```

Comment:
I thnink this should be in `sunpy/data/sample/__init__.py` so that it has to be explicitly imported.